src/bootstrap/config.rs

```diff
@@ -416,6 +416,7 @@ impl Config {
             config.docdir = install.docdir.clone().map(PathBuf::from);
             config.bindir = install.bindir.clone().map(PathBuf::from);
             config.libdir = install.libdir.clone().map(PathBuf::from);
+            config.libdir_relative = install.libdir.clone().map(PathBuf::from);
```

Comment:
Hm so actually I think this may be an old vestige that just hasn't actually been removed. Could the `libdir-relative` key get removed in favor of inferring it from `libdir`?

Reply:
I've removed `libdir_relative` from `Config` struct.